misc/db-updates/2023-07-17/classic_data_objects.sql

```diff
@@ -0,0 +1,14 @@
+alter table itemCreators drop constraint itemCreators_ibfk_2;
+DROP table creators;
+
+DROP trigger if exists fki_itemCreators_libraryID;
+DROP trigger if exists fku_itemCreators_libraryID;
+
+CREATE TABLE `creators` (
+  `creatorID` bigint unsigned NOT NULL AUTO_INCREMENT,
+  `firstName` varchar(255) CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci DEFAULT NULL,
+  `lastName` varchar(255) CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci DEFAULT NULL,
+  `fieldMode` tinyint(1) unsigned DEFAULT NULL,
+  PRIMARY KEY (`creatorID`),
+  KEY `name` (`lastName`(7),`firstName`(6))
+) ENGINE=InnoDB DEFAULT CHARSET=utf8;
```

Comment:
This will need to be a PHP script like the others that does a proper migration of existing data. It should set the shard to `down`, wait 20 seconds, do the migration, and bring the shard up.

Reply:
Right. I only kept it as .sql until now for convenience.
I'll have it wrapped into the same algorithm as [this update](https://github.com/zotero/dataserver/blob/master/misc/db-updates/2022-11-05/convertItemIDToBigInt), except with state=`down`.